src/Motion.cpp

```diff
@@ -156,38 +158,43 @@ int Motion::init()
         if (cfg->motion.roi_1_x == IVS_AUTO_VALUE)
         {
             cfg->set<int>(getConfigPath("roi_1_x"), channelAttributes.encAttr.picWidth - 1, true);
+            cfg->motion.rois[0].p1_x = channelAttributes.encAttr.picWidth - 1;
```

Comment:
I didn't find a more elegant way to do it via cfg->set()... but thought I'd change roi_coud to IVS_AUTO_VALUE and check that.

```
if (cfg->motion.roi_count == IVS_AUTO_VALUE)  {
    cfg->motion.rois[0].p1_x = channelAttributes.encAttr.picWidth - 1;
    cfg->motion.rois[0].p1_x = channelAttributes.encAttr.picHeight - 1;
}
```

and remove all previous implementation as it is not used elsewhere.